packages/js/src/components/modals/editorModals/GooglePreviewModal.js

```diff
@@ -4,19 +4,31 @@ import { __ } from "@wordpress/i18n";
 /* Internal dependencies */
 import EditorModal from "../../../containers/EditorModal";
 import SnippetEditorWrapper from "../../../containers/SnippetEditor";
+import React from "react";
```

Comment:
Importing `React` in this file is restricted and this causes an error in the linting check. The import statement should be removed